articles/search/search-security-overview.md

```diff
@@ -166,16 +166,18 @@ Currently, the only external resource that a search service writes customer data
 
 ### Exceptions to data residency commitments
 
-Although customer data isn't stored outside of your region, object names (considered as customer data), will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Telemetry logs include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
+Non-personal data received from an organization, or generated from the use of the product, such as component names, will be stored and processed outside your selected region or location. This data will appear in the telemetry logs used by Microsoft to provide support for the service. Component names include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
 
 >[!IMPORTANT]
->Object names aren't obfuscated in the telemetry logs. If possible, please avoid using names that convey sensitive information.
+>Customers shouldn't place any personal data in those fields or create applications designed to store personal data in these fields.
```

Comment:
I deleted my previous comment for this line as well.  My concerns about this edit are 1) the phrase "these fields".  You haven't identified object names as fields yet, so it's not clear what you're referring tom and 2) references to "personal data".  This looks like you're saying that customers can't put personal data inside a field.  

Originally, the "important" note was alerting the customer that object names weren't obfuscated (normally, PII is obfuscated per GDPR), and the point it was trying to make was a new one.  Now, post-edits, it's just restating the previous paragraph.  I'm not sure what purpose the "important" block is fulfilling?

Reply:
How about:

Customers shouldn't place any sensitive data in these fields or create applications designed to store sensitive data in these fields.

Reply:
Can you say "name fields" instead of "fields"?   Or maybe "Because object names appear in telemetry logs, avoid disclosing sensitive information in the names you assign to indexes, fields, and other named objects in your search service."

Reply:
As Mike mentioned, we can replace the important note and change the introduction with:

Object names will be stored and processed outside of your selected region or location. Customers shouldn't place any sensitive data in name fields or create applications designed to store sensitive data in these fields. This data will appear in the telemetry logs used by Microsoft to provide support for the service. Object names include names of indexes, indexers, data sources, skillsets, containers, and key vault store.

Reply:
I like the text -- can you make the edits Angie?  It's your PR, so updates to the edited version go through you...